dirs/dirs.go

```diff
@@ -190,6 +198,36 @@ func init() {
 	SetRootDir(root)
 }
 
+// SnapHomeDirs returns a slice of the home directories currently in use.
+func SnapHomeDirs() []string {
+	snapHomeDirsMu.Lock()
+	defer snapHomeDirsMu.Unlock()
+	dirs := make([]string, len(snapHomeDirs))
+	copy(dirs, snapHomeDirs)
+	return dirs
+}
+
+// SetSnapHomeDirs sets SnapHomeDirs to the user defined values and appends /home if needed
+// Homedir must be a comma separated list of the user defined home directories
+// If homedirs is empty, SnapHomeDirs will be a slice of length 1 containing "/home"
+func SetSnapHomeDirs(homedirs string) {
```

Comment:
this could return a []string

Reply:
if it does that it would need to behave like SnapHomeDirs in terms of the result